src/devices/bus/isa/areplay.h

```diff
@@ -0,0 +1,103 @@
+// license:BSD-3-Clause
+// copyright-holders:Bavarese
+/***************************************************************************
+
+	Action Replay for DOS (ISA card 1994; DATEL UK)
+
+***************************************************************************/
+
+#ifndef MAME_BUS_ISA_AREPLAY_H
+#define MAME_BUS_ISA_AREPLAY_H
+
+#pragma once
+
+#include "isa.h"
+#include "machine/timer.h"
+
+//**************************************************************************
+//  MACROS / CONSTANTS
+//**************************************************************************
+// One shot timer (555 timer with unknown interval)
+// TODO: determine exact value
+#define ONE_SHOT_TIMER_DELAY_MS 100
+
+#define SIZE_ROM_WINDOW 0x2000
+#define SIZE_RAM_BANK 0x1000
+
+
+// This is the RAM bank chosen when the AR starts up (after Reset):
+#define INITIAL_RAM_BANK_NO 1
+
+// Assume the scratch is at bank 0 in RAM (0 never selected by firmware, which only accesses 1 - 1F)
+#define SCRATCH_RAM_BANK_NO 0
+
+
+// Port 1:
+#define WRITE_ENABLE_BIT  0x01
+#define FREEZE_BUTTON_BIT 0x02
+#define ACTION_REPLAY_IRQ 0x08
+#define SLOMO_SWITCH_BIT  0x40
+#define LED_ENABLE_BIT    0x80
```

Comment:
You can't pollute the global namespace in a header like this. Preprocessor macros are global and can't be scoped.  This will affect anything that includes this header, including other headers included after it.  Find a better way of doing it.